From 2e732a34165f61510bc39b2f986279e02c4fd9d9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Viktor=20Sj=C3=B6lind?= <viktor.sjolind@pelagicore.com>
Date: Tue, 1 Aug 2017 12:37:55 +0200
Subject: [PATCH] Build SimpleNodeStateMachine
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>
---
 Makefile.am  | 2 +-
 configure.ac | 4 ++++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 6fbdd8b..ec60a12 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -22,4 +22,4 @@ ACLOCAL_AMFLAGS=-I m4
 SUBDIRS = NodeStateAccess @NSMC@ NodeStateManager
 
 check:
-	./run_tests.sh
\ No newline at end of file
+	./run_tests.sh
diff --git a/configure.ac b/configure.ac
index 50165fe..65b7788 100644
--- a/configure.ac
+++ b/configure.ac
@@ -96,4 +96,8 @@ AC_CONFIG_FILES([Makefile
                  NodeStateManager/config/nodestatemanager-daemon.service 
                  NodeStateManager/config/org.genivi.NodeStateManager.LifeCycleControl.service 
                  NodeStateManager/config/node-state-manager.pc])
+
+# SimpleStateMachine
+AC_CONFIG_SUBDIRS([SimpleNodeStateMachine])
+
 AC_OUTPUT
-- 
2.13.2

